### ISSUE TITLE
Systemd Journal parser

### ISSUE THREAD

**Issue by Zawadidone** [fox-it/dissect.target#161]
Systemd Journal parser

## References

[https://wiki.archlinux.org/title/Systemd/Journal](https://wiki.archlinux.org/title/Systemd/Journal)
[https://github.com/log2timeline/plaso/blob/main/plaso/parsers/systemd_journal.py](https://github.com/log2timeline/plaso/blob/main/plaso/parsers/systemd_journal.py)

**Comment by Schamper** [comment]
We have some code for this lying around, let me know if you intend to work on this yourself. I can dig it up to use as starting point.

**Comment by Zawadidone** [comment]
Yes, maybe a colleague of mine will work on this, but I'm not sure
